src/test/robot/api/storage/pravega.robot

```diff
@@ -36,7 +37,7 @@ Create Event Stream Test
 
 Create Byte Streams Test
     [Tags]  create_byte_streams
-    ${node_addr} =  Get Environment Variable  SERVICE_HOST  127.0.0.1
```

Comment:
Why did you use the ip instead of SERVICE_HOST variable?

Reply:
I brought back the SERVICE_HOST variable, but set it to localhost as it's the ip we use to connect to pravega that now runs in docker